programmers/아이디-추천.js

```diff
@@ -0,0 +1,12 @@
+function solution(newId) {
+    newId = newId.toLowerCase()
+        .replace(/[^\w-_.]/g, '')
```

Comment:
```suggestion
        .replace(/[^\w-.]/g, '')
```
\w에는 문자, 숫자, _가 포함되어 있어, _는 따로 기입하지 않으셔도 됩니다!